staff/tomas-sancheznavarro/js-arrays/shift.js

```diff
@@ -0,0 +1,23 @@
+function shift(arr) {
+
+    if (typeof arr === 'undefined') throw Error('no parameter has been introduced');
+    if (!(arr instanceof Array)) throw Error('input is not array');
```

Comment:
I think the error message on line 4 absorbs the error on line 3

Reply:
Correction applied! Got rid of redundant throw Error, thanks!